quipucords/quipucords/settings.py

```diff
@@ -56,7 +56,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(os.environ.get('DJANGO_DEBUG', True))
 
```

Comment:
Doesn't have to be this PR but should the default be `False` ?

Reply:
Its false in the container environment: https://github.com/quipucords/quipucords/blob/master/Dockerfile#L58

But allows dev to run with it set to True.

Reply:
ah cool